client/src/component/index.jsx

```diff
@@ -27,35 +27,36 @@ class App extends React.Component {
 
     getRandomImageId() {
         const min = 0;
-        const max = 3;
+        const max = 4;
         return Math.floor(Math.random() * (max - min)) + min;
     }
 
     Search(event) {
         // Render only if user types something in the input field
-        if (this.state.value) {
-            var search = this.state.value;
-            ReactDOM.render(<Nav celebrity={search}/>, document.getElementById("app"));
-            event.preventDefault();
+        if (this.state.searchValue) {
+            var search = this.state.searchValue;
+            ReactDOM.render(<Nav celebrity={search}/>, document.getElementById("root"));
         }
+       event.preventDefault()
     }
 
     handleInputChange(event) {
-        this.setState({value: event.target.value });
+        this.setState({searchValue:event.target.value});
+        event.preventDefault();
     }
 
     render() {
         const images = [
-            "/client/src/BTS1.jpg",
-            "/client/src/BTS2.jpeg",
-            "/client/src/IZ*ONE1.jpg",
-            "/client/src/BTS4.jpg",
+            process.env.PUBLIC_URL+"/img/redvelvet1.jpg",
+            process.env.PUBLIC_URL+"/img/BTS.jpg",
+            process.env.PUBLIC_URL+"/img/IZONE.jpg",
+            process.env.PUBLIC_URL+"/img/twice.jpg"
```

Comment:
I think that `%PUBLIC_URL%` should just work

Reply:
%PUBLIC_URL% only works on .html files
JS requires process.env infront